app.js

```diff
@@ -11,8 +11,11 @@ mongoose.connection.once("open", () => {
   console.log("connected to database");
 });
 
+const cors = require("cors");
 const app = express();
 
+app.use(cors());
+
```

Comment:
Great job remembering and adding cors, I think this will solve the issue I'm having making requests with the Apollo Client, where I get a [Network error]: TypeError: Failed to fetch.